.github/workflows/cd.yml

```diff
@@ -0,0 +1,118 @@
+name: Continuous Delivery - Operator
```

Comment:
@mpryc I have tested the job in my fork, example of created PR https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/2690